README.md

```diff
@@ -1 +1,151 @@
 # ampersand-multifield-view
+
+## purpose
+A view module for intelligently grouping multiple field views. Works well with [ampersand-form-view][ampersand-form-view].
```

Comment:
"A view module for intelligently applying an hierarchy to form data.

`MultifieldView` gathers field views into a collection that can be treated as a single field view whose `value` is an object with keys and values corresponding to its sub-views' names and their values."

the bit about having its own template and logic could be included too